tasks/main.yml

```diff
@@ -12,6 +12,7 @@
     dest: "{{ adminer_install_dir }}/{{ adminer_install_filename }}"
     mode: 0644
     timeout: 60
+    force: true
```

Comment:
The concern I have here is that this basically forces a download of adminer every time this role runs (even if the version hasn't changed).

I'd rather have a check for the version (if that's possible?) like this one: https://github.com/geerlingguy/ansible-role-docker/blob/master/tasks/docker-compose.yml#L2-L6 — then if the version is different than latest, or configured, it would be downloaded at the newer version.

Just forcing the updated download in any case (even if the user is happy with the current version) could cause issues (at worst), and would also cause this task to always try downloading (at best).

At a minimum, maybe setting this `true` to a variable like `adminer_force_download_new_version: true` in defaults/main.yml (and documenting that in the README) is enough.